controllers/jobs.js

```diff
@@ -137,11 +177,11 @@ exports.updateJob = asyncHandler(async (req, res, next) => {
 });
 
 /**
- * @desc    Delete single job
- * @route   DELETE /api/jobs/:id
- * @access  Owner/Admin
+ * @desc    Deactivate (Soft delete) single job
+ * @route   PUT /api/jobs/:id/deactivate
+ * @access  Admin/Owner
  */
-exports.deleteJob = asyncHandler(async (req, res, next) => {
+exports.deactivateJob = asyncHandler(async (req, res, next) => {
   // check if job exists
   const job = await db.oneOrNone(
     'SELECT * FROM jobs WHERE job_id = $1',
```

Comment:
```suggestion
  const job = await db.one(
    'SELECT * FROM jobs WHERE job_id = $1',
```
and remove the `400` response below it